--- a/node_modules/handlebars/lib/index.js
+++ b/node_modules/handlebars/lib/index.js
@@ -15,12 +15,12 @@
 var base = require('./handlebars/base');
 
 // Each of these augment the Handlebars object. No need to setup here.
-// // istanbul ignore next
-// if (!require.extensions) {
-//   require.extensions = {};
-// }
-// istanbul ignore next
-// hack for browserify, an extension that allows loading of files with hbs extension
-// remove this explicit check for 'Handlebars', this should work in all environments that support CommonJS
-if (typeof Handlebars === 'undefined' && !require.extensions['.hbs']) {
+// // // istanbul ignore next
+// // if (!require.extensions) { // XXX: Patch: Comment out require.extensions
+// //   require.extensions = {};
+// // }
+// // istanbul ignore next
+// // hack for browserify, an extension that allows loading of files with hbs extension
+// // remove this explicit check for 'Handlebars', this should work in all environments that support CommonJS
+if (typeof Handlebars === 'undefined' /*&& !require.extensions['.hbs']*/) { // XXX: Patch: Comment out .hbs check
   require('./handlebars/compiler/code-gen');
   require('./handlebars/compiler/compiler');
   require('./handlebars/compiler/javascript-compiler');
